src/components/eip3643/user/MeaningOfLife.tsx

```diff
@@ -44,6 +46,21 @@ export default function MeaningOfLife() {
       >
         Send
       </Button>
+
+      <Button
+        onClick={async () => {
+          if (walletInterface === null) return null;
```

Comment:
when `walletInterface` can be null? maybe we need an additional hook to wait until the interface becomes ready and only then enable/disable buttons? not to have this check in onclicks every time